src/utils.js

```diff
@@ -31,24 +31,45 @@ class HtmlElementTypeError extends Error {
       // eslint-disable-next-line babel/new-cap
       `${context.utils.RECEIVED_COLOR(
         'received',
-      )} value must be an HTMLElement or an SVGElement.`,
+      )} value must ${expectedString}.`,
       withType,
     ].join('\n')
   }
 }
 
-function checkHasWindow(htmlElement, ...args) {
+class HtmlElementTypeError extends GenericTypeError {
+  constructor(...args) {
+    super('be an HTMLElement or an SVGElement', ...args)
+  }
+}
+
+class NodeTypeError extends GenericTypeError {
+  constructor(...args) {
+    super('be a Node', ...args)
+  }
+}
+
+function checkHasWindow(htmlElement, ErrorClass, ...args) {
```

Comment:
I think you could've kept the signature of this one intact as we only call it internally in this module, only once, and always to check for the HTML element check, and not the Node check.

Not a big deal or a blocking change request. Can stay as is. Maybe I'm missing something.

Reply:
See line 63, I actually call it for the node check ;-)

Reply:
yes, I know.

My point was that that was the only call to it. But never mind, it's ok as is. It was a bit of a nit-pick, and maybe a wrong one at that anyway.

Reply:
ah yeah I see what you mean :-) yeah, maybe it's a bit over-designed :-)